src/pages/components/datepicker.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Datepicker
+description: >
+  A Datepicker lets users select a date by showing them a calendar. It works in combination with a text input.
+package: '@zendeskgarden/react-datepickers'
+components:
+  - datepickers/src/elements/Datepicker/Datepicker.tsx
+  - datepickers/src/elements/DatepickerRange/DatepickerRange.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To select a single date, use a Datepicker</li>
+      <li>To select a start and an end date, use a Datepicker range</li>
+    </ul>
+  </Use>
+</Usage>
+
+## How to use it
+
+### Default
+
+The typical usage of a Datepicker component.
+
+import Default from '../../examples/datepicker/DatepickerDefault.tsx';
+import DefaultCode from '!!raw-loader!../../examples/datepicker/DatepickerDefault.tsx';
+
+<CodeExample code={DefaultCode}>
+  <Default />
+</CodeExample>
+
+### Custom date format
+
+Dates are formatted using the
+[Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat)
+utility. For other formats, consider libraries like
+[momentjs](https://momentjs.com/) or [date-fns](https://date-fns.org/).
```

Comment:
Recommend switching the order of `date-fns` and `momentjs` throughout, to slightly emphasize Garden's preference.

Reply:
Also, momentjs just entered "I'm not dead _yet_" mode https://momentjs.com/docs/